README.md

```diff
@@ -2,8 +2,16 @@
 
 This repository is a work in progress for running a TIMS terminology server. This is a demo setup for being able to simply clone this repo to your machine and then run:
 
-* bin/start-ts.sh  # to start the server
-* bin/stop-ts.sh  # to stop it.
-* bin/reset-ts.sh  # to rest to a clean checkout to start over.
+* `bin/start-ts.sh`  # to start the server
+* `bin/stop-ts.sh`  # to stop it.
+* `bin/reset-ts.sh`  # to rest to a clean checkout to start over.
 
 See [this doc](https://docs.google.com/document/d/1J7exF73Z0Z8W-SCYRrQ5qNQVMZTkIH0c4rf-sgjDBfk) for work in progress documentation and notes.
+
+## Workflows
+### Converting OWL
```

Comment:
@ShahimEssaid I know you plan on not running this directly, but importing the function and using your own CLI, but for now, here are some docs.

Reply:
removed in favor of docs PR and [google doc](https://docs.google.com/document/d/1J7exF73Z0Z8W-SCYRrQ5qNQVMZTkIH0c4rf-sgjDBfk/edit#)